src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -51,7 +51,7 @@ const ProfilePicture = ({
                   style={styles.profileImage(size, width)}
                   source={{
                     uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH,
-                    cache: 'cacheOnly'
+                    cache: 'web'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The cache property of the image source has been changed from `'cacheOnly'` to `'web'`. This means that images will now be fetched from the web every time instead of being loaded from the cache. This could potentially slow down the loading of images and increase data usage for users. If the intention was to ensure that the most recent profile picture is always displayed, consider using a hybrid approach where the cache is used first and if the image is not in the cache, then it is fetched from the web.
```diff
- cache: 'cacheOnly'
+ cache: 'web'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->